airflow/providers/apache/spark/operators/spark_jdbc.py

```diff
@@ -162,6 +165,25 @@ def __init__(
         self._hook: SparkJDBCHook | None = None
         self._use_krb5ccache = use_krb5ccache
 
+    @property
+    @deprecated(
+        reason="`_keytab` is deprecated and will be removed in the future. Please use `keytab` instead.",
```

Comment:
It is not strictly required to deprecate internal/private methods. We could just remove it